src/xcode/ENA/ENA/Source/Services/CoronaTestService/FamilyMemberAntigenTest.swift

```diff
@@ -0,0 +1,90 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct FamilyMemberAntigenTest: Equatable, Hashable {
+
+	// MARK: - Internal
+
+	var displayName: String
+
+	// The date of when the consent was provided by the tested person at the Point of Care.
+	var pointOfCareConsentDate: Date
+	// The date of when the test sample was collected.
+	var sampleCollectionDate: Date?
+	var registrationDate: Date
+	var registrationToken: String?
+	var qrCodeHash: String
+	var isNew: Bool
+
+	var testResult: TestResult
+	var finalTestResultReceivedDate: Date?
+	var testResultWasShown: Bool
+
+	var certificateSupportedByPointOfCare: Bool
+	var certificateConsentGiven: Bool
+	var certificateRequested: Bool
+
+	var uniqueCertificateIdentifier: String?
+
+	var isOutdated: Bool
+	var isLoading: Bool
+
+	var testDate: Date {
+		return sampleCollectionDate ?? pointOfCareConsentDate
+	}
+
+}
+
+extension FamilyMemberAntigenTest: Codable {
```

Comment:
why do we put this in an extension?

Reply:
I took that over from the `UserXXXTest` implementations. I think we implemented it like that so we do not have to implement the initializer explicitly. As `Codable` requires its own initializer we don't get the auto-generated member wise initializer anymore. But I added it manually now so we conform to our coding standards with protocol-conformances inside the types themselves.

Reply:
nice!